app/src/main/java/me/damonkelley/tictactoe_app/factory/PresetFactory.java

```diff
@@ -0,0 +1,44 @@
+package me.damonkelley.tictactoe_app.factory;
+
+import me.damonkelley.tictactoe_app.loop.LoopBuilder;
+import me.damonkelley.tictactoe_app.turn.AsyncComputerTurn;
+import me.damonkelley.tictactoe_app.turn.MultiPlayerHumanTurn;
+import me.damonkelley.tictactoe_app.turn.SinglePlayerHumanTurn;
+
+public class PresetFactory {
+
+    private static final String COMPUTER_VS_HUMAN = "computer-vs-human";
+    private static final String HUMAN_VS_COMPUTER = "human-vs-computer";
+    private static final String HUMAN_VS_HUMAN = "human-vs-human";
+
+    public static LoopBuilder presetFor(String preset) {
```

Comment:
`preset` is sort of a weird name as an argument here, and maybe even for the class in general - is there a more focused name you can think of that reflects the specific thing that you're pre-setting?